Sources/PrimerSDK/Classes/Core/Primer/Primer.swift

```diff
@@ -244,11 +246,11 @@ public class Primer {
         Analytics.Service.record(events: [sdkEvent, connectivityEvent, timingEvent])
         
         if let clientToken = clientToken {
-            try? ClientTokenService.storeClientToken(clientToken)
+            let _ = try? ClientTokenService.storeClientToken(clientToken)
         }
         
         presentingViewController = viewController
-        show(flow: .defaultWithVault)
+        show(flow: .default)
```

Comment:
I know naming is not good here, unfortunately that's what we have. The vault manager flow is the `defaultWithVault`, so this shouldn't be changed here since this is the `showVaultManager` function